_content/lorawan/frequencies-by-country.md

```diff
@@ -111,7 +111,7 @@ For discussions about the frequency plans in different countries, see the posts
 | Gabon
 | Gambia
 | Georgia | unknown, limited CEPT | CEPT Rec. 70-03 |
-| Germany | EU863-870<br />EU433 | [Non-specific Short Range Devices (SRD) regulations](https://www.bundesnetzagentur.de/SharedDocs/Downloads/DE/Sachgebiete/Telekommunikation/Unternehmen_Institutionen/Frequenzen/Allgemeinzuteilungen/2014_69_SRD_pdf.pdf?__blob=publicationFile&v=1), CEPT Rec. 70-03 |
+| Germany | EU863-870<br />EU433 | [Non-specific Short Range Devices (SRD) regulations](http://www.bnetza.de/SharedDocs/Downloads/DE/Sachgebiete/Telekommunikation/Unternehmen_Institutionen/Frequenzen/Allgemeinzuteilungen/2018_05_SRD_pdf.html), CEPT Rec. 70-03 |
```

Comment:
Reidrects to https://www.bundesnetzagentur.de/SharedDocs/Downloads/DE/Sachgebiete/Telekommunikation/Unternehmen_Institutionen/Frequenzen/Allgemeinzuteilungen/2018_05_SRD_pdf.html; any reason to use non-secure and short domain name?

Reply:
don't know - the url was told to me by Pressestelle@BNetzA.DE